common/src/main/java/me/lucko/luckperms/common/command/CommandManager.java

```diff
@@ -157,7 +158,12 @@ public boolean hasPermissionForAny(Sender sender) {
 
     private CommandResult execute(Sender sender, String label, List<String> arguments) {
         applyConvenienceAliases(arguments, true);
-
+        if(this.plugin.getConfiguration().get(ConfigKeys.BLOCK_NON_PLAYER_SOURCES)) {
+            if(sender.isConsole() || sender.isValid()) {
```

Comment:
Codestyle - should be a space after `if`.

Also, `sender.isValid()` will return true in most cases - it returns false if the player has since logged out and their player instance has been GCed.